### PR TITLE
[service.subtitles.titlovi] 1.1.3

### LINKED ISSUE #636: [service.subtitles.titlovi] v1.1.3
### Add-on details:

- **General**
  - Add-on name: Titlovi.com
  - Add-on ID: service.subtitles.titlovi
  - Version number: 1.1.3
  - Kodi/repository version: Gotham

- **Code location**
  - URL: https://github.com/mikimac/service.subtitles.titlovi
  - Revision/tag: bd357db  
  - Branch: master

### Checklist:

- [X ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### PR DESCRIPTION
**Pull request by Razzeee**
Closes #636

### CODE REVIEW

File: service.subtitles.titlovi/resources/lib/ti_utilities.py [hunk 1]
```diff
@@ -173,11 +175,30 @@ def search_subtitles(self, name, tvshow, season, episode, lang, year):
...
-        WebSock = urllib.urlopen(url) # Opens a 'Socket' to URL
-        htmlContent = WebSock.read() # Reads Contents of URL and saves to Variable
-        WebSock.close() # Closes connection to url
+        try:
+            useragent = {'User-Agent': "Mozilla/5.0"}
+            req = urllib2.Request(url, headers=useragent)
+            website = urllib2.urlopen(req)
+        except urllib2.URLError, e:
+            if hasattr(e, 'reason'):
+                print 'We failed to reach a server.'
```
Comment (Rechi) [inline service.subtitles.titlovi/resources/lib/ti_utilities.py]: replace print here and in all other places with `xbmc.log(...)`

File: service.subtitles.titlovi/resources/lib/ti_utilities.py [hunk 2]
```diff
@@ -173,11 +175,30 @@ def search_subtitles(self, name, tvshow, season, episode, lang, year):
...
 
     def openUrl(self, url):
-        WebSock = urllib.urlopen(url) # Opens a 'Socket' to URL
-        htmlContent = WebSock.read() # Reads Contents of URL and saves to Variable
-        WebSock.close() # Closes connection to url
+        try:
+            useragent = {'User-Agent': "Mozilla/5.0"}
+            req = urllib2.Request(url, headers=useragent)
+            website = urllib2.urlopen(req)
+        except urllib2.URLError, e:
```
Comment (Rechi) [inline service.subtitles.titlovi/resources/lib/ti_utilities.py]: `except ... as ...` is the preferred syntax as this is python3 compatible.

File: service.subtitles.titlovi/service.py [hunk 3]
```diff
@@ -52,6 +52,34 @@
...
+def openURLAdress(url, postData=None):
+    try:
+        useragent = {'User-Agent': "Mozilla/5.0"}
+        req = urllib2.Request(url, headers=useragent)
+        if postData:
+            data = urllib.urlencode(postData)
+            website = urllib2.urlopen(req, data)
+        else:
+            website = urllib2.urlopen(req)
+    except urllib2.URLError, e:
```
Comment (Rechi) [inline service.subtitles.titlovi/service.py]: `except ... as ...` is the preferred syntax as this is python3 compatible.

**Comment by MartijnKaijser** [comment]
I think this one is obsolete now there's an "official" version already in repo.

**Pull request closed without merging**